packages/features/bookings/Booker/components/BookEventForm/BookingFields.tsx

```diff
@@ -90,28 +90,10 @@ export const BookingFields = ({
           field.options = options.filter(
             (location): location is NonNullable<(typeof options)[number]> => !!location
           );
-          // If we have only one option and it has an input, we don't show the field label because Option name acts as label.
```

Comment:
As this logic was generic enough to be part of FormBuilder, it is moved to FormBuilderField component